apps/wallet/src/shared/accounts.ts

```diff
@@ -6,7 +6,9 @@ import { type CoinBalance } from '@iota/iota-sdk/client';
 export interface AddressFromFinder {
     publicKey: string;
     bipPath: Bip44Path;
-    // TODO: extend this balance in the future to include eg staking, vesting schedules, assets, ...
+    hasTimelockedObject: boolean;
+    hasMigrationObject: boolean;
+    hasAsset: boolean;
```

Comment:
nit, rename to stardust and also even though we fetch only 1 is more robust naming it plural imo
```suggestion
    hasTimelockedObjects: boolean;
    hasStardustObjects: boolean;
    hasAssets: boolean;
```